pom.xml

```diff
@@ -67,8 +67,8 @@ LICENSE file.
 
     <!-- datastore binding versions, lex sorted -->
     <accumulo.1.6.version>1.6.6</accumulo.1.6.version>
-    <accumulo.1.7.version>1.7.3</accumulo.1.7.version>
-    <accumulo.1.8.version>1.8.1</accumulo.1.8.version>
+    <accumulo.1.7.version>1.7.4</accumulo.1.7.version>
+    <accumulo.1.8.version>1.9.1</accumulo.1.8.version>
```

Comment:
just to confirm, the 1.9.1 client works with 1.8.0 and 1.8.1 servers?

Reply:
Yes, it should, and vice-versa. But, users should definitely *NOT* be using 1.8.x servers. Versions 1.8.0, 1.8.1, and 1.9.0 are subject to a series of critical data loss bugs that were fixed in 1.9.1.

Version 1.9.x is basically a continuation of 1.8.x maintenance line, but with some new superficial client APIs (superficial, as in, does not change class path requirements or RPC). Semver required a version bump, but it's basically 1.8.